crates/pop-cli/src/commands/call/contract.rs

```diff
@@ -596,7 +596,7 @@ impl Default for CallContractCommand {
 			value: DEFAULT_PAYABLE_VALUE.to_string(),
 			gas_limit: None,
 			proof_size: None,
-			url: url::Url::parse("wss://rpc1.paseo.popnetwork.xyz").unwrap(),
+			url: url::Url::parse("wss://rpc2.paseo.popnetwork.xyz").unwrap(),
```

Comment:
As above. Also, why not use the constant.

Reply:
Refactor to use constant https://github.com/r0gue-io/pop-cli/pull/536/commits/a5fe96e5650839c5398252b0ddf80540a87a0a4a